docs/HISTORY.txt

```diff
@@ -5,6 +5,8 @@ Changelog
 2.7.14 (unreleased)
 -------------------
 
+- Improve extraction worker. [mbaechtold]
```

Comment:
I think we are actually fixing an error and should indicate that in the changelog.
Maybe `Fix a bug when extraction worker is faster than ZODB transaction.`